easybuild/easyconfigs/d/DFA/DFA-0.3.4-GCCcore-9.3.0-Python-3.8.2.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'PythonBundle'
+
+name = 'DFA'
+version = '0.3.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/mvcisback/dfa'
+description = "Python library for modeling DFAs, Moore Machines, and Transition Systems."
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+builddependencies = [
+    ('poetry', '1.0.9', versionsuffix),
```

Comment:
@lexming Since you're using `GCCcore` as toolchain, `binutils` should be included as a build dep too

Reply:
fixed in 295f2d3